chainerx_cc/scripts/run-clang-format.sh

```diff
@@ -11,8 +11,13 @@ set -eu
 script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 root_dir="$(realpath "$script_dir"/..)"
 
+expected_clang_format_version=6.0
 
-clang_format=clang-format
+if which clang-format-${expected_clang_format_version} > /dev/null ; then
```

Comment:
I found that this [StackOverflow post](https://stackoverflow.com/a/677212) recommends using `command -v` instead of `which`.

```suggestion
if command -v clang-format-${expected_clang_format_version} >/dev/null 2>&1 ; then
```